src/arduino-timer.h

```diff
@@ -91,7 +91,7 @@ class Timer {
     {
         struct task * const t = static_cast<struct task * const>(task);
 
-        if (t) {
+        if (t && (tasks <= t) && (t < tasks + max_tasks) && (t->handler)) {
```

Comment:
lets remove the necessary extra `(` `)`

Reply:
I can do that.  It may take another day or so, as I am having a busy week.

Reply:
Done.